### DISCUSSION TITLE
limit recognize to a specific user?

### DISCUSSION THREAD

**Discussion by rhatguy** [nextcloud/recognize#311] [Q&A] [answered]
Is there a way to limit recognize to only scan files of a single user or directory of a user?  I've thousands of pictures in my library and I'd like to do some testing against a small data set before I turn the app loose on everything.

**Comment by ftrentini** [comment]
Additionally, I want to choose which user can or cannot use this feature.

**Comment by marcelklehr** [comment, marked as answer]
This is currently not possible, but will likely be added in the future. see #2 and #389

**Reply by DavidMndz** [reply]
Would it be easier instead of limit the usage per user, use the 'Limit for groups' available in Nextcloud? I have seen this for other Nextcloud apps, if we could get this I think it would resolve the problem.

**Reply by DavidMndz** [reply]
I have seen that Recognize app already has the option for "Limit for groups" at Nextcloud level, but I am not sure if that does the job indeed.

Edit: I saw it but after trying to enable it (and fail) the option now disapeared.
